src/app/system/manage-scheduler-jobs/view-scheduler-job/view-scheduler-job.component.scss

```diff
@@ -0,0 +1,3 @@
+a{
+    color: blue;
```

Comment:
```suggestion
 color: blue;
```

Reply:
Should I align straight vertically down to the 'a' tag?

Reply:
no just reduce indent a bit

Reply:
Adjusted the alignment